services/httpd/handler.go

```diff
@@ -662,9 +661,9 @@ func gzipFilter(inner http.Handler) http.Handler {
 
 // versionHeader taks a HTTP handler and returns a HTTP handler
 // and adds the X-INFLUXBD-VERSION header to outgoing responses.
-func versionHeader(inner http.Handler, version string) http.Handler {
+func versionHeader(inner http.Handler, h *Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Add("X-InfluxDB-Version", version)
+		w.Header().Add("X-InfluxDB-Version", h.Version)
```

Comment:
Related to the above comment, same thing.  Restricting scope for arguments is important.